README.md

```diff
@@ -88,6 +107,11 @@ those specified in the mapping. Each of the entries in the config provide mappin
 associated web relay input states and every RFPath defined in the sensor definition json
 file should have an entry in the preselector config. The keys in the dictionary may use the
 name of the RFPath or the index of the RFPath in the RFPaths array.
+The `sensors`, `digital_inputs`, and `analog_inputs` keys define the sensors,
```

Comment:
What's the difference between `sensors` and `analog_inputs`?

Reply:
I see now that `sensors` refer to 1-Wire sensors. Up to you if you'd like to add a bit more clarity here in the README, given that this set of 3 options is specific to ControlByWeb relays.